Cargo.toml

```diff
@@ -24,3 +24,4 @@ gcc = "0"
 [features]
 cuda = []
 tesseract = []
+nonfree = []
```

Comment:
There is no such concept as nonfree. You can try cfg under contrib flag: https://github.com/opencv/opencv_contrib.

I believe all of them all nonfree.

If they don't match, then ok, but still need some better naming, I believe.

Reply:
Sure, a better name would be appropriate.
Original problem was the fact that Ubuntu package of OpenCV seems to leave out some of the modules used by this package.
I just assumed they were not free enough for Debian.

I'm new to this, and would be happy with any other solution as well to make this build on Ubuntu.

There seems to be some interdependencies with tesseract to be resolved also.

Reply:
Well, [this script](https://github.com/nebgnahz/cv-rs/blob/master/.ci/travis_build_opencv.sh) works on travis which is using Ubuntu image. So if it's your problem then you can try to adopt it for you.

However, better separation could be a good thing. Just some thoughts.

Reply:
Thanks for the link. I see that builds OpenCV from source.
While that would enable the complete functionality and be a viable option on Ubuntu for some, I would need to be able to use the Ubuntu packaged version. This calls for a mechanism to disable some of the dependencies in the wrapper package. At least optionally, if not by default.

Is this general approach of feature configuration the right way to do this? If only the name(s) were more appropriate?
Where is the current set of feature names derived from? It's not 1:1 OpenCV module names either.

Reply:
@peahonen no, it's not only naming thing.

For example, I don't see why `mser` should be cutted out with cfg. There is no reason why it couldn't work on ubuntu. Can you confirm that when you download prebuilt package it doesn't contain MSER implementation?